tools/cbmc/include/cbmc.h

```diff
@@ -24,8 +27,10 @@
   * the maximum offset into an object in CBMC, and hence a bound on the
   * size of objects in CBMC.
   */
-#define CBMC_BITS               7
-#define CBMC_MAX_OBJECT_SIZE    ( 0xFFFFFFFF >> ( CBMC_BITS + 1 ) )
+#ifndef CBMC_OBJECT_BITS
+#define CBMC_OBJECT_BITS        7
+#endif
+#define CBMC_MAX_OBJECT_SIZE    ( 0xFFFFFFFF >> ( CBMC_OBJECT_BITS + 1 ) )
```

Comment:
Didn't you show to me that something like this is defined by default in a cbmc header file?  Is this a duplication?

Reply:
This is the same file. I only added the `#ifndef CBMC_OBJECT_BITS` around `#define CBMC_OBJECT_BITS` during our Chime call.